### PR TITLE
Feature/23 submit transaction

### PR DESCRIPTION
**Pull request by landgenoot**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -32,6 +32,7 @@ dependencies {
         exclude module: 'recyclerview-v7'
     }
     androidTestCompile 'org.mockito:mockito-android:2.7.22'
+    testCompile 'org.mockito:mockito-android:2.7.22'
```
Comment (jonathanraes) [inline app/build.gradle:35]: duplicate dependency, also line 52

File: app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import org.bitcoinj.core.listeners.DownloadProgressTracker;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+/**
+ * Created by daan on 9-6-17.
+ */
+
+public class ConfirmationTracker extends DownloadProgressTracker {
```
Comment (klikooo) [inline app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java]: Is this class used?

File: app/src/main/java/com/digitalvotingpass/blockchain/BlockChain.java [hunk 3]
```diff
@@ -121,4 +130,66 @@ public boolean assetExists(Asset asset) {
...
+        return transactions;
+    }
+
+    /**
+     * Create a new transaction, signes it with the travel document.
+     * @param utxo
+     * @param destination
+     * @param pcon
+     */
+    public byte[] getSpendUtxoTransaction(TransactionOutput utxo, Address destination, PassportConnection pcon) {
```
Comment (jonathanraes) [inline app/src/main/java/com/digitalvotingpass/blockchain/BlockChain.java]: This creates a new signed transaction, right? Method name is kind of unclear

File: app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java (hunk 2 above)
Comment (jonathanraes) [inline app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java]: Unused class

**Review by jonathanraes — CHANGES_REQUESTED** [review]
great, but listening for rejects is still good to do I think, since it can still happen, also timeout for confirmation

File: app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java (hunk 2 above)
Comment (landgenoot) [reply app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java]: Solved.

File: app/src/main/java/com/digitalvotingpass/blockchain/BlockChain.java (hunk 3 above)
Comment (landgenoot) [reply app/src/main/java/com/digitalvotingpass/blockchain/BlockChain.java]: Renamed it to `utxoToSignedTransaction()`

File: app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java (hunk 2 above)
Comment (landgenoot) [reply app/src/main/java/com/digitalvotingpass/blockchain/ConfirmationTracker.java]: Solved

File: app/build.gradle (hunk 1 above)
Comment (landgenoot) [reply app/build.gradle:35]: Solved

**Review by jonathanraes — APPROVED** [review]

**Pull request merged**